pilot/lib/pilot/options.rb

```diff
@@ -71,7 +71,7 @@ def self.available_options
         FastlaneCore::ConfigItem.new(key: :skip_waiting_for_build_processing,
                                      short_option: "-z",
                                      env_name: "PILOT_SKIP_WAITING_FOR_BUILD_PROCESSING",
-                                     description: "Don't wait for the build to process. If set to true, the changelog won't be set, `distribute_external` option won't work and no build will be distributed to testers",
+                                     description: "Don't wait for the build to process. If set to true, the changelog won't be set, `distribute_external` option won't work and no build will be distributed to testers. \n Uploading and verifying submissions to TestFlight can take sometime, it may use up unexpected minutes in your allotted CI plans. Use skip_waiting_for_build_processing: true to reduce used minutes.",
```

Comment:
Hmm, CI doesn't like that line length:

```
[16:42:10]: ▸ pilot/lib/pilot/options.rb:74:371: C: Metrics/LineLength: Line is too long. [418/370]
[16:42:10]: ▸ description: "Don't wait for the build to process. If set to true, the changelog won't be set, `distribute_external` option won't work and no build will be distributed to testers. \n Uploading and verifying submissions to TestFlight can take sometime, it may use up unexpected minutes in your allotted CI plans. Use skip_waiting_for_build_processing: true to reduce used minutes.",
[16:42:10]: ▸ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

I wonder how we can fix this... @joshdholtz @KrauseFx ?

Reply:
# rubocop:disable Metrics/LineLength
        👍 @janpio 
# rubocop:enable Metrics/LineLength

Reply:
That surely is one way :)